admin/app/com/lucidchart/piezo/admin/utils/CronHelper.scala

```diff
@@ -0,0 +1,185 @@
+package com.lucidchart.piezo.admin.utils
+
+import java.util.{Date, TimeZone}
+import org.quartz.{CronExpression, CronScheduleBuilder, CronTrigger, TriggerBuilder}
+import play.api.Logging
+import scala.util.control.NonFatal
+
+object CronHelper extends Logging {
+  val DEFAULT_MAX_INTERVAL = 0
+  val NUM_COMPLEX_SAMPLES = 8000
+
+  /**
+   * Determines the max interval for a cron expression. The operation fails silently because it's not crucial.
+   * @param cronExpression
+   */
+  def getMaxInterval(cronExpression: CronExpression): Long = getMaxInterval(cronExpression, None)
+  def getMaxInterval(cronExpression: CronExpression, timeZone: Option[TimeZone]): Long = {
```

Comment:
Yes, in the test file. This is also necessary because if someone was running the tests in a different timezone that didn't observe daylight savings their test results would fail since the trigger would use the default timezone.